app/main.py

```diff
@@ -1,6 +1,19 @@
-from typing import Callable
+from typing import Callable, Any
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}
+
+    def wrapper(*args, **kwargs) -> Any:
```

Comment:
Good to add an annotation, that this is a wrapped function.
```suggestion
    @wraps(func)
    def wrapper(*args, **kwargs) -> Any:
```